rest/rest-server/src/main/java/org/ow2/proactive_grid_cloud_portal/common/SessionStore.java

```diff
@@ -115,4 +118,19 @@ public int terminateExpiredSessions(long timeoutDelay) {
     public int size() {
         return sessions.size();
     }
+
+    public String renewSession(String oldSessionId) throws NotConnectedException {
+        Session oldSession = sessions.remove(oldSessionId);
+
+        if (oldSession == null) {
+            return oldSessionId;
+        }
+
+        String newSessionId = SessionIdGenerator.newSessionId();
```

Comment:
I don't understand why a new sessionId must be created as we are renewing the lease of an existing rest session